tfsec-to-trivy-migration-guide.md

```diff
@@ -0,0 +1,48 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes some differences between Trivy and tfsec.
+
+| Feature              | Trivy                                                  | tfsec                |
+|----------------------|--------------------------------------------------------|----------------------|
+| Policy Distribution | Embedded and Updated via Registry                      | Embedded             |
+| Custom Policies      | Rego                                                   | Rego, JSON, and YAML |
+| Supported Formats    | Dockerfile, JSON, YAML, Terraform, CloudFormation etc. | Terraform  Only      |
```

Comment:
need to remember to keep this up to date. I think this is already lacking, maybe link to trivy docs to keep it updated

Reply:
then we would reference tfsec in the Trivy docs, I thought we did not want to do that?

Reply:
ah